topology/Pittsburgh Supercomputing Center/PSC - Bridges/Bridges.yaml

```
@@ -35,3 +35,29 @@ Resources:
         Description: PSC-Bridges Hosted CE
         Details:
           hidden: false
+  PSC-Bridges:
+    ID: 1081
+    Disable: false
```

```suggestion
    Active: true
```

`Disabled` is deprecated in favor of`Active`